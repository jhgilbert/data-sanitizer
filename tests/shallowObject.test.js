const { sanitizeFields } = require("..")

test('sanitizes a shallow object', () => {
  let data = {
    id: 1,
    name: 'Jen'
  }

  const sanitationResult = sanitizeFields({ data, fieldNames: ['id'] })
  expect(sanitationResult).toMatchSnapshot()
})