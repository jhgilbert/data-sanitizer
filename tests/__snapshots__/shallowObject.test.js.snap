// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sanitizes a plain object 1`] = `
Object {
  "data": Object {
    "id": "[SANITIZED]",
    "name": "Jen Gilbert",
  },
  "removedValues": Object {
    "id": Array [
      1,
    ],
  },
}
`;

exports[`sanitizes a shallow object 1`] = `
Object {
  "data": Object {
    "id": "[SANITIZED]",
    "name": "Jen Gilbert",
  },
  "removedValues": Object {
    "id": Array [
      1,
    ],
  },
}
`;
